Cargo.toml

```diff
@@ -20,18 +20,18 @@ fxhash = "0.2"
 log = "0.4"
 num-traits = "0.2"
 spirv = { package = "spirv_headers", version = "1.5", optional = true }
-pomelo = { version = "0.1.4", optional = true }
 thiserror = "1.0.21"
 serde = { version = "1.0", features = ["derive"], optional = true }
 petgraph = { version ="0.5", optional = true }
 rose_tree = { version ="0.2", optional = true }
-pp-rs = { version = "0.1", optional = true }
+# TODO: Make another release of glslpp-rs
```

Comment:
it's OK to depend on a git revision (but needs either a particular revision, or a tag) until we need to publish Naga itself

Reply:
I'm trying to get the branch merged so that we can keep this pointing to the main repo or potentially a new release, also I tried to add a rev but cargo complained because of the branch